ui/library/src/components/BoundingBox.tsx

```diff
@@ -38,12 +41,16 @@ export const BoundingBox: React.FunctionComponent<Props> = ({
   }, [pageDimensions, rotation, scale]);
 
   return (
-    <div
-      id={id}
-      className={componentClassName}
-      style={getBoundingBoxStyle()}
-      onClick={onClick}
-      {...extraProps}
-    />
+    <React.Fragment>
+      <div
+        className="reader__page-over__bounding-box-highlighted-border"
+        style={getBoundingBoxStyle()}></div>
```

Comment:
Also I think you are missing `lay` in your `className`. The class name `reader__page-overlay__bounding-box-highlighted` and `reader__page-overlay__bounding-box-highlighted-border` not following BEM rules. I would probably name it as `reader__page-overlay__bounding-box-bottomline` since it is always rendered no matter highlighted or not. (But better ask Paul's opinion. I think the naming here is kind of complicated)

Reply:
@yensung  Paul was the one whom suggested the original name when we were discussing it

Reply:
Well then the class name should be `reader__page-overlay__bounding-box-highlighted-border` not `reader__page-over__bounding-box-highlighted-border`.

Reply:
@yensung i decided to name its underline since its acutally an underline

Reply:
Man I was meaning that you were missing `lay` behind `over`
<img width="676" alt="image" src="https://user-images.githubusercontent.com/84034381/163019413-523a173e-b390-4828-88db-e31de0500d9d.png">

Reply:
@yensung i couldnt find this one which file is it??